4-reduction/utils.py

```diff
@@ -21,11 +21,15 @@ def fetch_data(name: str, quiet=True) -> str:
     logger = pooch.get_logger()
     logger.setLevel("ERROR" if quiet else "INFO")
 
-    file_path = pooch.retrieve(
-        url=f"https://public.esss.dk/groups/scipp/dmsc-summer-school/2025/{name}.zip",
-        known_hash=None,
-        processor=pooch.Unzip(),
+    registry = pooch.create(
+        path=pooch.os_create('dmsc_school'),
```

Comment:
Some things are still wrong: `os_create` doesn't seem to exist.

Reply:
Thanks! I should have just run the notebooks locally